src/ReflectionClosure.php

```diff
@@ -36,6 +36,15 @@ class ReflectionClosure extends ReflectionFunction
      */
     public function __construct(Closure $closure, $code = null)
     {
+        // Not sure where to put this
```

Comment:
I think these definitions can be moved before the class definition.

Reply:
Awesome idea. Done!